.github/workflows/codeql-analysis.yaml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   codeql:
-    name: Compute Code Coverage
+    name: Run CodeQL Security Vulnerability Analysis
```

Comment:
I accidentally committed an "add a suggestion" and wrote the wrong thing - corrected